.dockerignore

```diff
@@ -32,3 +32,4 @@ pip-selfcheck.json
 *~
 **/tests/
 **/tests_azure/
+/tools/c7n_kube/.skaffold
```

Comment:
leading slash should be omitted for ./

note you have .skaffold content in this pr.

Reply:
The files in `.skaffold` don't need to be in the docker image, but they are useful. 

Does the leading `/` not work the same way in `.dockerignore` as it does in `.gitignore`, ie "ignore this file as identified by this absolute path, rooted where the docker context starts"?

Reply:
I think I missed this was a docker ignore file.